src/po/actions/scenarios.actions.js

```diff
@@ -81,24 +84,25 @@ export const userFilteringCardByLabel = async () => {
   await basepage.TrelloComponents.item("filter").click();
   await Promise.all(basepage.TrelloComponents.urgentCards.map(async card => {
     const isDisplayed = await card.isDisplayed();
-    expect(isDisplayed).toBe(true);
+    assert.isTrue(isDisplayed);
   }));
   await Promise.all(basepage.TrelloComponents.nonUrgentCards.map(async card => {
     const isDisplayed = await card.isDisplayed();
-    expect(isDisplayed).toBe(false);
+    assert.isFalse(isDisplayed);
```

Comment:
add aditional message to assert to make clear in report for example what really happens
for e.g. assert.isFalse(isDisplayed, 'Non urgent cards is displayed');